src/Components/Votes.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react"
+import { patchVotes } from "./Api"
+
+const Votes = ({ votes, article_id }) => {
+
+    const [updateVotes, setUpdatedVotes] = useState(votes)
+
+    const handleClick = (incVotes) => {
+        setUpdatedVotes((currentVotes) => {
+            return currentVotes + incVotes
+        })
+        patchVotes(article_id, incVotes)
+        .catch(() => {
+            setUpdatedVotes((currentVotes) => {
+                return currentVotes - incVotes
+            })
+        })
+    }
```

Comment:
This is nice and clean 👏 It's doable without making a new state for `updatedVotes` because it can all be done using the `articleByID` state if you wanted